apps/ledger-live-desktop/tests/page/speculos.page.ts

```diff
@@ -49,99 +60,59 @@ export class SpeculosPage extends AppPage {
     await pressBoth();
   }
 
-  @step("Send method - EVM")
-  async sendEVM(tx: Transaction) {
-    const events = await pressUntilTextFound(DeviceLabels.ACCEPT);
-    const isAmountCorrect = containsSubstringInEvent(tx.amount, events);
-    expect(isAmountCorrect).toBeTruthy();
-    const isAddressCorrect = containsSubstringInEvent(tx.accountToCredit.address, events);
-    expect(isAddressCorrect).toBeTruthy();
-
-    await pressBoth();
-  }
-
-  @step("Send method - BTC-based coin")
-  async sendBTCBasedCoin(tx: Transaction) {
-    let deviceLabel: string;
-
-    switch (tx.accountToDebit.currency) {
-      case Currency.BTC:
-        deviceLabel = DeviceLabels.CONTINUE;
-        break;
-      case Currency.DOGE:
-        deviceLabel = DeviceLabels.ACCEPT;
-        break;
-      default:
-        throw new Error(`Not a BTC-based coin: ${tx.accountToDebit.currency}`);
-    }
-
-    const events = await pressUntilTextFound(deviceLabel);
-    const isAmountCorrect = containsSubstringInEvent(tx.amount, events);
-    expect(isAmountCorrect).toBeTruthy();
-    const isAddressCorrect = containsSubstringInEvent(tx.accountToCredit.address, events);
-    expect(isAddressCorrect).toBeTruthy();
-    await pressBoth();
-    await waitFor(DeviceLabels.CONFIRM);
-    await pressUntilTextFound(DeviceLabels.ACCEPT);
-    await pressBoth();
-  }
-
   @step("Sign Send Transaction")
   async signSendTransaction(tx: Transaction) {
     const currencyName = tx.accountToDebit.currency;
     switch (currencyName) {
       case Currency.sepETH:
-        await this.sendEVM(tx);
+      case Currency.POL:
+        await sendEVM(tx);
         break;
-      case Currency.BTC:
       case Currency.DOGE:
```

Comment:
is this in use ? there's no instruction in `case`

Reply:
In this case, the absence of an instruction in the first case allows it to "fall through" to the next case. This means that both Currency.BTC and Currency.DOGE will execute the same block of code. It's a way to handle multiple conditions with a single instruction